<launch>

  <!--  Arm Controllers -->
  <rosparam command="load" file="$(find pr2_mannequin_mode)/pr2_arm_controllers_loose.yaml" />
  <node pkg="pr2_controller_manager" type="spawner" name="spawn_arm_controllers"
        args="--stopped r_arm_controller_loose l_arm_controller_loose" />

</launch>
